.github/actions/run-integration-tests/action.yml

```diff
@@ -1,40 +1,18 @@
 name: "Run integration tests"
 description: "Run Inspektor Gadget integration tests"
 
-inputs:
-  registry:
-    description: 'Server address of registry where integration tests container image will be pushed'
-    required: false
-  container_repo:
-    description: 'The repository where integration tests container image will be pushed'
-    required: true
-  image_tag:
-    description: 'The container image tag which will be used as integration tests container image tag'
-    required: true
-  username:
-    description: 'Username used to log against the registry'
-    required: false
-  password:
-    description: 'Password or access token used to log against the registry'
-    required: false
-
 runs:
   using: "composite"
   steps:
     - name: Get kubectl-gadget-linux-amd64.tar.gz from artifact.
       uses: actions/download-artifact@v2
       with:
         name: kubectl-gadget-linux-amd64-tar-gz
-        path: /home/runner/work/inspektor-gadget/
+        path: /home/runner/work/inspektor-gadget/inspektor-gadget/
```

Comment:
Just so that the binary downloaded or built locally are at the same location as expected [here](https://github.com/kinvolk/inspektor-gadget/blob/main/Makefile#L165).